qiskit/circuit/library/__init__.py

```diff
@@ -385,6 +385,18 @@
 NCT (Not-CNOT-Toffoli) template circuits
 ----------------------------------------
 
+Template library
+--------------------------
+
+Existing Template circuits.
+
+.. autosummary::
+   :toctree: ../stubs/
+
+   TemplateLibrary
+
+----------------------------------------
+
 Template circuits for :class:`~qiskit.circuit.library.XGate`,
 :class:`~qiskit.circuit.library.CXGate`,
 and :class:`~qiskit.circuit.library.CCXGate` (Toffoli) gates.
```

Comment:
This paragraph belongs to the `NCT` title. Maybe you can introduce your section after it?